dev-haskell/clash-cores/clash-cores-1.8.1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.4.0
+# manually udpdated to use the GitHub repo instead of hackage
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal git-r3
+
+DESCRIPTION="A collection of IP cores for Clash"
+HOMEPAGE="https://github.com/clash-lang/clash-compiler"
+
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/clash-lang/clash-compiler"
+EGIT_COMMIT="v1.8.1"
+S="${S}/${PN}"
+CABAL_FILE="${S}/${CABAL_PN}.cabal"
+
+LICENSE="BSD-2"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="+doctests haddock nix +unittests"
+
+RDEPEND="dev-haskell/clash-lib:=[profile?]
+	dev-haskell/clash-prelude:=[profile?]
+	dev-haskell/constraints:=[profile?]
+	>=dev-haskell/ghc-typelits-extra-0.3.2:=[profile?]
+	>=dev-haskell/ghc-typelits-knownnat-0.6:=[profile?]
+	>=dev-haskell/ghc-typelits-natnormalise-0.6:=[profile?]
+	>=dev-haskell/infinite-list-0.1:=[profile?] <dev-haskell/infinite-list-0.2:=[profile?]
+	dev-haskell/lens:=[profile?]
+	dev-haskell/pretty-show:=[profile?]
+	>=dev-haskell/prettyprinter-1.2.0.1:=[profile?] <dev-haskell/prettyprinter-1.8:=[profile?]
+	>=dev-haskell/prettyprinter-interp-0.2:=[profile?] <dev-haskell/prettyprinter-interp-0.3:=[profile?]
+	dev-haskell/quickcheck:=[profile?]
+	>=dev-haskell/reducers-3.12.2:=[profile?] <dev-haskell/reducers-4.0:=[profile?]
+	>=dev-haskell/string-interpolate-0.3:=[profile?] <dev-haskell/string-interpolate-0.4:=[profile?]
+	>=dev-haskell/text-1.2.2:=[profile?] <dev-haskell/text-2.2:=[profile?]
+	>=dev-lang/ghc-8.8.1:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-3.0.0.0
+	doctests? ( unittests? ( test? ( dev-haskell/clash-prelude-hedgehog
+						>=dev-haskell/doctest-parallel-0.2 <dev-haskell/doctest-parallel-0.4
```

Comment:
This package is masked

https://github.com/gentoo-haskell/gentoo-haskell/blob/7f612790664de2a7499b411490f85382ffaa5300/profiles/package.mask#L64-L68

You'll have to find a way to remove it here and on `clash-lib` as well.

It looks like the easiest way to do this would be to disable the `doctests` flag, which you can do when you generate the ebuild by passing `--flags=-doctests` to `hackport`.

Reply:
Thanks, I've updated it.